src/util/urlHelpers.js

```diff
@@ -19,6 +19,7 @@ export const PROFILE_PAGE_PENDING_APPROVAL_VARIANT = 'pending-approval';
 
 // No access page - path params:
 export const NO_ACCESS_PAGE_POST_LISTINGS = 'posting-right';
+export const NO_ACCESS_PAGE_INITIATE_TRANSACTIONS = 'transaction-right';
```

Comment:
So, the URL would be /no-transaction-right
This is a user facing string, so good to think if there are something more understandable.
(But personally, I couldn't think anything better...)

Reply:
I asked Katri's opinion on this in the same wireframe comment where I'm double checking the copy for the no-access page.

Reply:
URLs updated to use plural for both posting and transaction rights, i.e.
- /no-posting-rights
- /no-transaction-rights